session.go

```diff
@@ -0,0 +1,113 @@
+package scylla
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"scylla-go-driver/transport"
+)
+
+// TODO: Should we support connecting to different than default port?
+// TODO: Add retry policy.
+// TODO: Add Query Paging.
+// TODO: make better randomConnection.
+
+type SessionConfig struct {
+	hosts []string
+	transport.ConnConfig
+}
+
+func MakeSessionConfig(tcpNoDelay bool, defaultConsistency uint16, hosts ...string) SessionConfig {
```

Comment:
Config object are constructed manually as structs, please remove.
One thing to note here is making the default (zero) values useful.